scoring/score.py

```diff
@@ -1,20 +1,64 @@
+# -*- coding: utf-8 -*-
+
+import itertools
 import collections
 
 
 class InvalidScoresheetException(Exception):
     pass
 
 
+POINTS_HELD = 1
```

Comment:
Should we use the terminology "in control" in line with the rules?

Reply:
Hrm, it would end up being `tokens_in_control_of_robot` which is rather verbose.